CODE_OF_CONDUCT.rst

```diff
@@ -40,11 +40,11 @@ As members of the community,
   same respect as people within our community.
 
 * We pledge to help the entire community follow the code of conduct, and to
-  not remain silent when we see violations of the code of conduct. We will
-  take action when members of our community violate this code such as
-  contacting einsteinpy.project@gmail.com (all emails sent to this
-  address will be treated with the strictest confidence) or talking privately
-  with the person.
+  not remain silent when we see violations of the code of conduct. Action will
+  be taken against the members of our community who violate the code which 
+  includes: contacting shreyas@einsteinpy.org (all emails sent to this address 
+  will be treated with the strictest confidence) or talking privately with the 
+  person.
```

Comment:
I don't think my exact words were these!

It was and I quote you from our personal chat (https://matrix.to/#/!SKrapYYjQQMEQEWAmC:matrix.org/$11R1dulC6QbVVBemO0S1WBcYrY3H0Fo4VEPWhos2XDk?via=matrix.org):

```
We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. We will take action when members of our community violate this code such as contacting shreyas@einsteinpy.org (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.
```